base/interactiveutil.jl

```diff
@@ -598,7 +598,7 @@ else
                 rethrow()
             end
         elseif downloadcmd == :curl
-            run(`curl -L -f -o $filename $url`)
+            run(`curl -L -f -s -o $filename $url`)
```

Comment:
Yeah I think that will require more serious thinking about the api etc.  I think for now we should at least try to make the experience consistent; dumping out the progress bar  from curl directly doesn't seem very Julian. I agree it would be useful to have some control over the progress etc.